tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from metaworld.envs.mujoco.env_dict import ALL_ENVIRONMENTS, ALL_V2_ENVIRONMENTS
+from metaworld.policies import *
+from tests.metaworld.envs.mujoco.sawyer_xyz.utils import check_success
+
+test_data_v1 = []
+
+test_data_v2 = [
+    # name,      policy,      success rate,   environment kwargs
+    ['push-wall-v2', SawyerPushWallV2Policy(), 0.8, {}]
+]
+
```

Comment:
Instead of splitting tests into v1 and v2, it may be better to have one big list. @avnishn thoughts?

Reply:
if we Union the 2 dicts before creating the environments we can make everything one list pretty quickly

Reply:
this is a pretty small detail. I think we should just stick with this format unless anyone can see a reason why not to use it/if there is a format that provides some significant improvement.